src/Model/GroundWaterFlow/gwf3sto8.f90

```diff
@@ -339,15 +390,15 @@ subroutine sto_fc(this, kiter, hold, hnew, njasln, amat, idxglo, rhs)
           if (this%iconf_ss == 0) then
             zold = bt + DHALF * tthk * snold
             znew = bt + DHALF * tthk * snnew
-            rhsterm = -rho1 * (snold * (hold(n) - zold) + snnew * znew)
+            rhsterm = -rho1old * snold * (hold(n) - zold) - rho1 * snnew * znew
           else
-            rhsterm = -rho1 * ssh0 + rho1 * ssh1
+            rhsterm = -rho1old * ssh0 - rho1 * ssh1
```

Comment:
This is ok. 

Taking another look at the way I implemented this option I think it is confusing. 

I think the variable names should be modified. I think it would be easier to follow if the variable `ssh0` was changed to `ssb0` and set equal to `hold - top` when `hold > tp` and `0` otherwise. Similarly, `ssh1` should probably just be `tp` and `rho1` should be multiplied by `ss_sat1` like `aterm` is. So it would look like

`rhsterm = -rho1old * ssb0 - rho1 * ss_sat1 * tp`

@damianmerrick I can take care of this refactoring after the PR gets merged. Or feel free to make the change.